src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -5,8 +5,9 @@ Release History
 
 2.0.19
 ++++++
-* Call "agent" a "node" in AKS to match documentation.
-* Deprecate --orchestrator-release option in acs create
+* call "agent" a "node" in AKS to match documentation
+* deprecate --orchestrator-release option in acs create
```

Comment:
not so sure whether the change entries here are related, but fine if it is a catch-up for other merged PRs